src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector.md

```diff
@@ -14,15 +14,388 @@ tree_item_index: 1100
+        }
+    ]
+}
+```
+
+Make sure that all configuration values are set in the `nuxeo.conf`:
+
+```
+nuxeo.s3storage.transient.bucket=
+nuxeo.aws.accessKeyId=
+nuxeo.aws.secretKey=
+nuxeo.aws.region=
+nuxeo.s3storage.bucket=
+```
+{{#> callout type='note'}}
+
+Since the work in [NXP-29548](https://jira.nuxeo.com/browse/NXP-29548) 'nuxeo.s3storage.useDirectUpload=true' and 'nuxeo.s3storage.transient.roleArn=' (documented as part of the previous version configuration) are no longer necessary **for the Nuxeo Aspera Connector** and can be removed, **if you only use these values for the Nuxeo Aspera Connector, and not for any other reason**.  See upgrade notes.
+{{/callout}}
+
+#### Aspera Nuxeo Configuration
+
+Add the aspera access keys in `nuxeo.conf`:
+
+**Example**:
+
+```
+## is the same for the 2 nodes
+aspera.node.url=https://ats-aws-us-east-1.aspera.io
+aspera.node.port=443
+
+### ACCESS KEY ON UPLOAD NODE LINKED TO TS S3 BUCKET
+aspera.access.key.id=
+### SECRET KEY ON UPLOAD NODE LINKED TO TS S3 BUCKET
+aspera.access.key.secret=
+
+### ACCESS KEY ON DOWNLOAD NODE LINKED TO TMAIN S3 BUCKET
+aspera.download.access.key.id=
+### SECRET KEY ON DOWNLOAD NODE LINKED TO MAIN S3 BUCKET
+aspera.download.access.key.secret=
+```
+
+### Functional Overview
+The **Nuxeo Aspera Connector** uses **Transfers** to add content to the Nuxeo system.  Transfers are “wrappers” or “buckets” for the files/folders you add to the Nuxeo system.
+
+#### Transfer steps
+The process of adding content is divided into 2 steps: **Upload** and **Create**. 
+In the first step, the user **uploads** content to the transfer bucket. While in this step, it may be possible, depending on the chosen “transition type”,  for a user to add files to the bucket and add properties that will apply to the content. 
+Once content and any metadata have been added, the transfer can be completed and the content is then **created** in the system.
+
+#### Transfer transition types
+Transitioning between the “upload” and “create” steps can be achieved in 2 ways: **automatically (auto-create)** or **manually**. Users can choose which option to use when creating the transfer from the transfer dashboard.   It is possible to change from manual to auto-create after you start adding content to the transfer.  **Once you set the to auto-create, however, the transfer will behave as if you started in that way, and some limitations around modifying the transfer will apply**.
+
+When auto-create is toggled/selected to be “on”, the user needs only to start the upload of the folders/files to the transfer.  Once the upload is completed, the system will automatically start the process to create the content in the system; no need to wait around simply to push a button.  **Before adding content** to the transfer, you can edit the **“common metadata”** which would apply to all files in the transfer.  Users can, of course, decide not to include any metadata at all, and simply add files, too.
```

Comment:
For consistency purposes should we use `content` or `files and folders`?

```suggestion
When auto-create is toggled/selected to be “on”, the user needs only to start the upload of the content to the transfer.  Once the upload is completed, the system will automatically start the process to create the content in the system; no need to wait around simply to push a button.  **Before adding content** to the transfer, you can edit the **“common metadata”** which would apply to all files in the transfer.  Users can, of course, decide not to include any metadata at all, and simply add files, too.
```

Reply:
we had more places where it said "files" and "folders" so I kept that and changed references to "content" to read "files and folders".  Regardless of which option, I think we should now be using consistent terms (and will read through the rest of the doc with that in mind; this page as well as the upgrade/release notes.)